website/docs/guides/scala-js.md

```diff
@@ -39,6 +39,39 @@ scala-cli package Hello.scala --js --js-emit-source-maps
 #  node ./Hello.js
 ```
 
+## Scala.Js DOM support
```

Comment:
NIT
```suggestion
## Scala.js DOM support
```
(just realized I've been making a typo too, `Scala.JS` instead of `Scala.js` - but it's not `Scala.Js`. We should unify those at some point…)